src/public_data/publishing_datasets.md

```diff
@@ -0,0 +1,51 @@
+# Making Datasets Publicly Available
+
+Currently, only datasets and query results that are available in BigQuery and 
+defined in the [bigquery-etl][bigquery_etl] repository can be made publicly available. 
+See [Scheduling BigQuery Queries in Airflow](../cookbooks/bigquery-airflow.md)
+on how to create and schedule datasets. Before data can be published, a data review is
+required.
+
+To make query results publicly available, a [`metadata.yaml` file][bigquery_etl_metadata] 
+must be added alongside the query in [bigquery-etl][bigquery_etl]. For example:
+
+```yaml
+friendly_name: SSL Ratios
+description: >-
+  Percentages of page loads Firefox users have performed that were 
+  conducted over SSL broken down by country.
+owners:
+  - example@mozilla.com
+labels:
+  application: firefox
+  incremental: true     # incremental queries add data to existing tables
+  schedule: daily       # scheduled in Airflow to run daily
+  public_json: true
+  public_bigquery: true
+  review_bug: 1414839   # Bugzilla bug ID of data review
+  incremental_export: false  # non-incremental JSON export writes all data to a single location
+```
+
+The following options define how data is published:
+* `public_json`: data is available through the [public HTTP endpoint][public_data_endpoint]
+* `public_bigquery`: data is publicly available on BigQuery
```

Comment:
It's worth noting here that `public_bigquery` is a prerequisite for `public_json`. We can't publish json without also having the data publicly available in BQ, correct?

Reply:
It is currently possible to publish data as JSON even if it's not public in BigQuery